package.json

```diff
@@ -190,6 +190,7 @@
     "universal-analytics": "0.4.20",
     "update-check": "1.5.3",
     "utility-types": "2.1.0",
+    "whatwg-url": "7.0.0",
```

Comment:
Why exactly is this needed, instead of just using the native URL module?

Reply:
Native `URL` is not available in Node 8

Reply:
@TooTallNate really?

![image](https://user-images.githubusercontent.com/16088670/62068042-85d74800-b235-11e9-97d5-7db7547d72ba.png)

Reply:
Oh good catch. It's the _global_ `URL` object that is not available in Node 8, but we can import this one and use it 👍

Reply:
ebfbbd42